woodwork/column_accessor.py

```diff
@@ -85,13 +89,59 @@ def __eq__(self, other):
             return False
         return self._series.equals(other._series)
 
+    def __getattr__(self, attr):
+        '''
+            If the method is present on the Accessor, uses that method.
+            If the method is present on DataFrame, uses that method.
+        '''
+        if self._schema is None:
+            raise AttributeError("Woodwork not initialized for this Series. Initialize by calling Series.ww.init")
+        if hasattr(self._series, attr):
+            return self._make_series_call(attr)
+        else:
+            raise AttributeError(f"Woodwork has no attribute '{attr}'")
+
     def __repr__(self):
         msg = u"<Series: {} ".format(self._series.name)
         msg += u"(Physical Type = {}) ".format(self._series.dtype)
         msg += u"(Logical Type = {}) ".format(self.logical_type)
         msg += u"(Semantic Tags = {})>".format(self.semantic_tags)
         return msg
 
+    def _make_series_call(self, attr):
+        '''
+        Forwards the requested attribute onto the series object.
+        Intercepts return value, attempting to initialize Woodwork with the current schema
+        when a new Series is returned.
+        Confirms schema is still valid for the original Series.
+        '''
+        series_attr = getattr(self._series, attr)
+
+        if callable(series_attr):
+            def wrapper(*args, **kwargs):
+                # Make Series call and intercept the result
+                result = series_attr(*args, **kwargs)
+
+                # Try to initialize Woodwork with the existing Schema
+                if isinstance(result, pd.Series):
+                    if result.dtype == self._schema['logical_type'].pandas_dtype:
+                        schema = copy.deepcopy(self._schema)
+                        # We don't need to pass dtype from the schema to init
+                        _ = schema.pop('dtype')
+                        result.ww.init(**schema)
+                    else:
+                        invalid_schema_message = f'Operation performed by {attr} has invalidated the Woodwork ' \
+                            'typing information:\n' \
+                            f'dtype mismatch between original dtype, {self._schema["logical_type"].pandas_dtype}, ' \
+                            f'and returned dtype, {result.dtype}.\n' \
+                            'Please initialize Woodwork with Series.ww.init'
+                        warnings.warn(invalid_schema_message, TypingInfoMismatchWarning)
```

Comment:
Might not be worth changing the `get_warning_message` for this warning so that it works for both Series and Dataframe, but since the warning messages are otherwise the same, it'd be nice to have.

Reply:
Yeah, that thought crossed my mind as well. I think it should be a fairly simple change. I'll update.